docker/docker-compose.dev.stacks-krypton-2.1-transition.yml

```diff
@@ -0,0 +1,17 @@
+version: '3.7'
+services:
+  stacks-blockchain:
+    image: "zone117x/stacks-api-e2e:stacks2.1-transition-38c5623"
```

Comment:
Should we move this repo to hirosystems before release?

Reply:
Done!